components/reports/temperature/ReportTempChart.vue

```diff
@@ -168,12 +197,41 @@ export default {
 							width: 0
 						},
 						fillcolor: '#cccccc',
-						opacity: 0.2
+						opacity: 0.2,
 					},
 				],
 				hovermode: 'x unified',
+				annotations: [],
+				legend: {
+					x: 1.03
+				},
+				margin: {
+					b: 120
+				},
+				height: 500,
+			}
+
+			let footer_y = -0.4
+			if (window.innerWidth < 1250) {
+				layout['xaxis'] = {
+					tickangle: 45,
+				}
+				layout['margin']['b'] = 160
+				footer_y = -0.65
 			}
 
+			layout.annotations.push({
+				x: 0.5,
+				y: footer_y,
+				xref: 'paper',
+				yref: 'paper',
+				showarrow: false,
+				text: 'The boxplot represents the interquartile range (IQR) of historical means for the season, from 1950-2009.<br /> \
```

Comment:
This isn't centered in my preview (see screenshot in the general conversation about this ticket), which is confusing me!

Reply:
Fixed in 9d178c97.